torch/quantization/fx/quantization_patterns.py

```diff
@@ -1607,7 +1638,8 @@ def convert(self,
                 node_name_to_scope: Dict[str, Tuple[str, type]],
                 load_arg: Callable,
                 is_reference: bool = False,
-                convert_custom_config_dict: Dict[str, Any] = None) -> Node:
+                convert_custom_config_dict: Dict[str, Any] = None,
+                equalization_enabled: bool = False) -> Node:
```

Comment:
Do we need to add this argument to add the instances of `convert`, given that it has a default value?